model/src/main/scala/ch.epfl.scala.index.model/release/LanguageVersion.scala

```diff
@@ -72,12 +73,17 @@ object Scala3Version {
   def isValid(version: BinaryVersion): Boolean =
     version match {
       case MajorBinary(3) => true
+      case MinorBinary(3, _) => true
+      case PatchBinary(3, _, _) => true
```

Comment:
Isn't it always supposed to be `Major(3)` ("_3")?

Reply:
I don't know. This method is used to validate searches as well, so if I search for `3.0.1` binary artifacts I would like the validation to be true for all `3.x.y`

Reply:
Currently the `scalaVersion` field in the search API is the Scala binary version so it should be `2.13` or `3` rather than `2.13.5` or `3.0.1`.

Reply:
Can we accept any version and internally transform it to binary version? I don't know how many Scala developers are familiar with this concept and its evolution in Scala 3.
We already try to hide this information in `sbt` with `%%`, it seems reasonable to accept any version here

Reply:
I am not totally against it but I don't want to make this API more complex than it already is. In particular I think it is unfortunate that the fields of the search API are different form the fields of the search page and I think we should aim at merging them somehow. Your proposition goes in the opposite direction.

In the long run, I think we should design a new API that would be used by the front-end to load the different parts of the page asynchronously. Ideally this API should be as simple as possible (not smarter than what needs the scaladex website). Then we can also provide a nice library on top of it that can do the smart stuff (like computing the binary version).